drivers/clone.c

```diff
@@ -478,23 +498,11 @@ static int sstate_dead(int maxage)
 
 static int instcmd(const char *cmdname, const char *extra)
 {
-	const char	*val;
-
-	val = dstate_getinfo(getval("load.status"));
-	if (val) {
-		if (!strcasecmp(val, "off") || !strcasecmp(val, "no")) {
-			outlet = 0;
-		}
```

Comment:
Moved to `parse_args` where we have this information, if the outlet is off should not be set inside an instant command handling function, but where the data actually comes in and is parsed. This also obsoletes the need for the redundant dstate check which is expensive.